mypy/nodes.py

```diff
@@ -3235,10 +3235,26 @@ def enum_members(self) -> list[str]:
             name
             for name, sym in self.names.items()
             if (
-                isinstance(sym.node, Var)
-                and name not in EXCLUDED_ENUM_ATTRIBUTES
-                and not name.startswith("__")
-                and sym.node.has_explicit_value
+                (
+                    isinstance(sym.node, Var)
+                    and name not in EXCLUDED_ENUM_ATTRIBUTES
+                    and not name.startswith("__")
```

Comment:
Should this condition exist for enum.member too because https://github.com/python/mypy/issues/18563?

Reply:
Hm, good question. Let's keep it for now, I will change it in https://github.com/python/mypy/issues/18563 if needed.